content/2023/march-development-day: Development day summary (#2)
AaltoSciComp/blog: content/2023/march-development-day.rst

  position anyway.

* Continue the possibility of **on-demand courses** if someone
  requests them.

* The first two points above squeeze out medium-sized courses for the
  most part, in our opinion.  We'll still offer medium-sized courses
  when we see a need - if you want to request one, let us know!

We will make sure that we have good recommendations for advanced
self-study courses and generally chart out the resources so that our
users don't have to.  This is mostly done by our `Hands-on Scientific
Computing <https://hands-on.coderefinery.org>`__ course.

Triton v3
---------

Triton is our HPC cluster, and is notable for being a Ship of Theseus:
it's continually upgraded while being the same cluster.  This has
resulted in the software running it getting a bit out of date.  This
software was originally developed as broader partnerships, and as
these partnerships have changed, we need to take more responsibility
for it ourselves.

Users shouldn't see any major change from this, though part of it is
improving our (user) software installation tools, which should make
increased responsiveness to software installation requests.

LUMI
----

> As said above, LUMI is a significant resource, yet no one has ever
> come to use asking for help using it.  Over the past six months, we

Reviewer: ```suggestion
As said above, Lumi is a significant resource, yet our users have not
come to us asking for our help in using it. Over the past six months, we
```

have found some Triton users who would benefit from it and helped
extend their workflows to work on LUMI.  We do this by first testing
some applications ourselves, then looking at Triton usage for large
users and reaching out directly.

Currently our focus is on GPU-intensive applications, which is made
more interesting because LUMI has AMD GPUs.  We've gotten local AMD
GPUs for our own testing and in general are well prepared to support
this.

While LUMI is a HPC system and has a typical HPC system interface, it
serves so many different users that the software stack is very
limited, so that most users need to install their own software and
figure out how to run it on AMD GPUs.  This is why we recommend most users

> access LUMI through us (we're paid to save you time, after al), though

Reviewer: ```suggestion
access LUMI through us (we're paid to save you time, after all), though
```
